examples/2.0.0/aws-sns-publish.yml

```
@@ -0,0 +1,80 @@
+asyncapi: 2.0.0
+id: 'https://github.com/asyncapi/asyncapi/blob/master/examples/2.0.0/aws-sns-publish.yml'
+info:
+  title: 'Simple AWS SNS Publish specification'
+  description: |
+    Specification for a service that publishes various "hello world"
+    type messages to the helloWorld SNS topic
+  version: '0.1.0'
+servers:
+  development:
+    # This URL doesn't really describe a development server, but it does describe the
+    # top-level URL used to subscribe to messages from a given AWS region
+    url: 'https://sns.us-east-1.amazonaws.com/'
+    description: 'Development/test account for message publishing'
+    protocol: 'sns'
+    protocolVersion: '2010-03-31'
+    bindings:
+      # AccountID for this environment
+      awsAccountID: "123456789012"
```

What about putting the region here too? It's a feature of the server in the end:

```suggestion
      awsAccountID: "123456789012"
      region: "us-east-1"
```

We can do the same for the other servers.